client/src/parser.js

```diff
@@ -0,0 +1,99 @@
+const { parse } = require('path');
+
+/**
+ * @description Parses an JSON created from the .bpmn xml of the model to an executable .robot file
+ * @returns {string} Code that has to be put in .robot file
+ */
+
+// this should be refactored to a dictionary
+function taskLib(app) {
+  if (app == 'Excel.Files') return 'RPA.Excel.Files';
+  if (app == 'Browser') return 'RPA.Browser';
+}
+
+function parseDiagramJson(json_data) {
+  let parsedCode = '';
+  var bpmnTasks = json_data['bpmn2:definitions']['bpmn2:process']['bpmn2:task'];
+
+  // create Settings statements
+  parsedCode += '*** Settings ***\n';
+  parsedCode += 'Documentation  Our first parsed RPA\n';
+
+  if (bpmnTasks !== undefined) {
+    console.log('bpmnTasks: ' + bpmnTasks);
```

Comment:
I think when pushing we should always remove all ``Console.log()`` statements from the code.